tests/python/testing.py

```diff
@@ -1,6 +1,6 @@
 # coding: utf-8
 from xgboost.compat import SKLEARN_INSTALLED, PANDAS_INSTALLED, DT_INSTALLED
-from xgboost.compat import CUDF_INSTALLED, DASK_INSTALLED
+from xgboost.compat import CUDF_INSTALLED, CUPY_INSTALLED, DASK_INSTALLED
```

Comment:
Can we merge this after the release?  I would like to investigate lazy library loading.  These libraries are all heavy in size.

Reply:
Yep, I just measured it takes 0.6s to import cupy vs 1.73s for xgboost.